opm/input/eclipse/Schedule/Network/ExtNetwork.cpp

```diff
@@ -205,6 +212,36 @@ void ExtNetwork::update_node(Node node)
     this->m_nodes.insert_or_assign(name, std::move(node) );
 }
 
+bool ExtNetwork::needs_instantaneous_rates(const Opm::Schedule& schedule, const int report_step) const {
```

Comment:
Based on the usage of this function in https://github.com/OPM/opm-simulators/pull/5410, it basically indicates whether we are handling network flow with extended network with "NO" in the item 3 in WEFAC or GEFAC. 

Or more simply, we can use `true` for extended network and use `false` for standard network and group calculation. If we can introduce a flag to mark whether the network is extended network, we do not need this function anymore.  And `use_efficiency_in_ext_network` will take care whether we will use instantaneous rate. 

What do you think?  I can understand this part wrong. Please let me know.

Reply:
This function can certainly be skipped, but the intention is to save some computation in opm-simulators in the quite common situation with default value (YES) for all instances of WEFAC/GEFAC with wells/groups in the network. 

One complication I see is action handling (i.e., if item3 is updated through actions), will need to ensure the check is performed again in this case. Can have a quick look, or we just drop this for now.